sql/mocks/flow.go

```diff
@@ -0,0 +1,143 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
```

Comment:
Can we add the mockery command used to generate this mock part of the `make mock` command
Reference: https://github.com/astronomer/astro-cli/blob/main/Makefile#L23-L40